packages/commerce-sdk-react/src/hooks/ShopperCustomers/queryKeyHelpers.ts

```diff
@@ -126,49 +127,49 @@ export type QueryKeys = {
 // This is defined here, rather than `types.ts`, because it relies on `Client` and `QueryKeys`,
 // and making those generic would add too much complexity.
 type QueryKeyHelper<T extends keyof QueryKeys> = {
-    /**
-     * Reduces the given parameters (which may have additional, unknown properties) to an object
-     * containing *only* the properties required for an endpoint.
-     */
-    parameters: (params: Params<T>) => Params<T>
+    // /**
+    //  * Reduces the given parameters (which may have additional, unknown properties) to an object
+    //  * containing *only* the properties required for an endpoint.
+    //  */
+    // parameters: (params: Params<T>) => Params<T>
```

Comment:
[NIT] We should remove these comments for consistency across the other `queryKeyHelpers.ts` files